QUICK_START_GUIDE.md

```diff
@@ -2,135 +2,140 @@
 
 ## Setup OpenVINO™ Training Extensions
 
 1. Clone the training_extensions repository with the following commands:
 
-   ```
-   git clone https://github.com/openvinotoolkit/training_extensions.git
-   cd training_extensions
-   git checkout develop
-   git submodule update --init --recursive
+   ```bash
+   $ git clone https://github.com/openvinotoolkit/training_extensions.git
+   $ cd training_extensions
+   $ git checkout develop
+   $ git submodule update --init --recursive
    ```
 
-2. Install prerequisites with:
+1. Install prerequisites with:
 
-   ```
-   sudo apt-get install python3-pip python3-venv
+   ```bash
+   $ sudo apt-get install python3-pip python3-venv
+   # verify your python version
+   $ python3 --version; pip3 --version; virtualenv --version
+   Python 3.8.10
+   pip 20.0.2 from /usr/lib/python3/dist-packages/pip (python 3.8)
+   virtualenv 20.0.17 from /usr/lib/python3/dist-packages/virtualenv/__init__.py
    ```
 
-   Although they are not required, You may also want to use Jupyter notebooks or OTE CLI tools:
+   (Optional) You may also want to use Jupyter notebooks or OTE CLI tools:
 
    ```
-   pip3 install notebook; cd ote_cli/notebooks/; jupyter notebook
+   $ pip3 install notebook; cd ote_cli/notebooks/; jupyter notebook
    ```
 
-3. Search for available scripts that create python virtual environments for different task types:
+1. There available scripts that create python virtual environments for different task types:
 
    ```bash
-   find external/ -name init_venv.sh
+   $ find external/ -name init_venv.sh
    ```
 
-   Sample output:
+   > **_Note_** The following scripts are valid for the current version of the project
 
    ```
-   external/mmdetection/init_venv.sh
-   external/mmsegmentation/init_venv.sh
-   external/deep-object-reid/init_venv.sh
+   external/model-preparation-algorithm/init_venv.sh
+   external/anomaly/init_venv.sh
    ```
 
-   Each line in the output gives an `init_venv.sh` script that creates a virtual environment
-   for the corresponding task type.
-
-4. Choose a task type, for example,`external/mmdetection` for Object Detection.
+   - `external/model-preparation-algorithm/init_venv.sh` can be used to create a virtual environment for the following task types.
 
-   ```bash
-   TASK_ALGO_DIR=./external/mmdetection/
-   ```
+     - Classification
+     - Detection
+     - Segmantation
 
-   Note that the variable `TASK_ALGO_DIR` is set in this example for simplicity and will not be used in scripts.
+   - `external/anomaly/init_venv.sh` can be used to create a virtual environment for the following task types.
+     - Anomaly-classification
+     - Anomaly-detection
+     - Anomaly-segmentation
```

Comment:
Task type should follow the original TaskType enum in ote_sdk.entity.model_template.
Could you add i-seg / rotated det, and fix anomaly-cls -> anomaly_cls, etc?

Reply:
updated